### PR TITLE
D2D-014

### PR DESCRIPTION
**Pull request by HridoyMA**

### CODE REVIEW

File: utils/constants/common-constants.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { SidebarItem } from '@/models/global-types';
 import { HomeIcon, BookIcon, SettingIcon } from '../../assets/icons';
 
-const PAGE_ROUTES = {
+export const PAGE_ROUTES = {
   Dashboard: '/dashboard',
   Leads: '/leads',
   Settings: '/settings',
+  Signin: 'auth/signin',
```
Comment (tajnur007) [inline utils/constants/common-constants.ts:8]: I hope, it will be `/auth/signin`.

**Review by tajnur007 — COMMENTED** [review]
You have to discard the `yarn.lock` file changes.

**Comment by HridoyMA** [comment]
fixed

**Review by tajnur007 — APPROVED** [review]
PR Approved!

**Pull request merged**